addons/mrp/tests/test_stock_report.py

```diff
@@ -218,39 +218,39 @@ def test_kit_packaging_delivery_slip(self):
         superkit = self.env['product.product'].create({
             'name': 'Super Kit',
             'type': 'consu',
-            'packaging_ids': [(0, 0, {
+            'uom_id': self.env['uom.uom'].create({
                 'name': '6-pack',
-                'qty': 6,
-            })],
+                'relative_factor': 6,
+            }).id,
         })
 
         compo01, compo02 = self.env['product.product'].create([{
             'name': n,
             'is_storable': True,
             'uom_id': self.env.ref('uom.product_uom_meter').id,
-            'uom_po_id': self.env.ref('uom.product_uom_meter').id,
         } for n in ['Compo 01', 'Compo 02']])
 
         self.env['mrp.bom'].create({
             'product_tmpl_id': superkit.product_tmpl_id.id,
             'product_qty': 1,
             'type': 'phantom',
+            'product_uom_id': superkit.uom_id.id,
             'bom_line_ids': [
-                (0, 0, {'product_id': compo01.id, 'product_qty': 1}),
-                (0, 0, {'product_id': compo02.id, 'product_qty': 1}),
+                (0, 0, {'product_id': compo01.id, 'product_qty': 6}),
+                (0, 0, {'product_id': compo02.id, 'product_qty': 6}),
             ],
         })
 
-        for back_order, expected_vals in [('never', [12, 12, 2, 2]), ('always', [24, 12, 4, 2])]:
+        for back_order, expected_vals in [('never', [12, 12]), ('always', [24, 12])]:
             picking_form = Form(self.env['stock.picking'])
             picking_form.picking_type_id = self.picking_type_in
             picking_form.partner_id = self.partner
             with picking_form.move_ids_without_package.new() as move:
                 move.product_id = superkit
-                move.product_uom = self.env.ref('uom.product_uom_dozen')
-                move.product_uom_qty = 2
+                # move.product_uom = self.env.ref('uom.product_uom_pack_6')
```

Comment:
These comments should be removed (and yes, it's 100% me that left them there, sorry :smiling_face_with_tear:)